docs/modules/ROOT/pages/cli/getting_started_in_cli.adoc

```diff
@@ -0,0 +1,192 @@
+
+== Getting started in the CLI
+
+In addition to the Stonesoup user interface (UI), you can also use the command line interface (CLI) to access Stonesoup. Using the CLI can help you configure Stonesoup to meet your specific needs.
+
+.*Prerequisites*
+
+To access Stonesoup by using the CLI, you must configure oc commands. Follow the instructions in xref:https://docs.openshift.com/container-platform/4.11/cli_reference/openshift_cli/getting-started-cli.html[Getting started with the OpenShift CLI].
+
+.*Procedure* 
+
+To log in to Stonesoup by using the CLI, complete the following steps: 
+
+. Navigate to the xref:https://registration-service-toolchain-host-operator.apps.appstudio-stage.x99m.p1.openshiftapps.com/[OpenShift Developer Sandbox].
+. Select the checkbox next to *I agree to the Terms and Conditions*.
+. Select *Get started with OpenShift Developer Sandbox*.
+. On the top right of your screen, select *Proxy login command*.
+. Select the *copy icon* to copy the command and your unique token. 
+. In your terminal, paste the command that you copied in step 5. This command runs `oc login --token=[your token]`. 
+
+.*Verification*
+
+To verify that `oc login` worked correctly, complete the following steps:
+
+. Run `cd ~/.kube`.
+. Run `ls` and verify that there is a file named `config`. 
+
+== Useful oc commands for Stonesoup
+
+The following list provides some useful oc commands and examples of their outputs, if relevant:
+
+=== oc get application
+* Command: 
+
+`oc get application` 
```

Comment:
Thanks! I am not a fan of the current format, but I couldn't find a really good way to do it. I wanted to avoid a table. 

Is it stylistically ok to write only "Example output: {output}"? Wasn't sure if it was ok because it's not a full sentence 

But I can def omit the "command:"

Reply:
"Example output: {output}" for ones where you don't have example output?